src/drawer.js

```diff
@@ -247,13 +247,15 @@ $.Drawer.prototype = {
             var viewportSize = this._calculateCanvasSize();
             if( this.canvas.width != viewportSize.x ||
                 this.canvas.height != viewportSize.y ) {
+                var imageSmoothingEnabled = this.canvas.getContext('2d').imageSmoothingEnabled;
                 this.canvas.width = viewportSize.x;
                 this.canvas.height = viewportSize.y;
                 if ( this.sketchCanvas !== null ) {
                     var sketchCanvasSize = this._calculateSketchCanvasSize();
                     this.sketchCanvas.width = sketchCanvasSize.x;
                     this.sketchCanvas.height = sketchCanvasSize.y;
                 }
+                this.setImageSmoothingEnabled(imageSmoothingEnabled);
```

Comment:
It might be overkill, but I think I might actually split `setImageSmoothingEnabled` into two functions; a "private" one that just sets the various Canvas flags for different browsers, and the "public" one that sets `_imageSmoothingEnabled`, calls the "private" one, and tells the viewer to redraw. That way you can just call the "private" one from here, since you don't need the other stuff in this case. It could be called `_setImageSmoothingEnabled` or something.